### PR TITLE
Add element to diagram mapping to MDKzip

### LINKED ISSUE #22: Add element to diagram mapping to MDKzip
- [x] Environment option to enable/disable the export of the mapping
- [x] for each diagram there should be one mapping file *.mdkmapping within the MDKzip
- [x] the mapping file should be a json file with the following structure
  - `diagram { id: Diagram.ID, representedElementsRecursively: [ ModelObject.ID ] }`
- [x] Extend svg export to have transparent objects on the svg images referring to ModelObject.IDs

### PR DESCRIPTION
**Pull request by MooK9**
closes #22 
- Added environment option to toggle element mapping extraction
- Text is selectable on the exported diagrams

### CODE REVIEW

File: src/main/java/gov/nasa/jpl/mbee/mdk/fileexport/FileExportRunner.java [hunk 1]
```diff
@@ -193,6 +198,7 @@ private void writeDataToZip(ProgressStatus progressStatus, ElementContextProvide
 			writeData(progressStatus, contextProvider, elements, filePrinter);
 		}
 	}
+
```
Comment (debrecenics) [inline src/main/java/gov/nasa/jpl/mbee/mdk/fileexport/FileExportRunner.java:201]: Typo: diagramPresetntationElements (however, this is not part of the PR, but please, fix it).

File: src/main/java/gov/nasa/jpl/mbee/mdk/fileexport/FileExportRunner.java [hunk 2]
```diff
@@ -306,10 +340,17 @@ private void exportDiagramsToZip(Collection<DiagramPresentationElement> diagrams
 					zipOutputStream.write(bytes, 0, length);
 				}
 			}
-		diagramFile.deleteOnExit();
+			options.getGeneralOptions().setUseSVGTextTag(useSvg);
```
Comment (debrecenics) [inline src/main/java/gov/nasa/jpl/mbee/mdk/fileexport/FileExportRunner.java:343]: Similarly to the previous comment. This should be moved outside of the while cycle.

**Review by bergmanngabor — APPROVED** [review]
